pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -49,9 +49,7 @@ var (
 	getNetInterfaceAddrsFunc        = getNetInterfaceAddrs
 	createHnsEndpointFunc           = createHnsEndpoint
 	getNamespaceEndpointIDsFunc     = hcn.GetNamespaceEndpointIds
```

Comment:
didn't you remove the only usage of `getNamespaceEndpointIDsFunc`? We can probably remove it now